lib/elastic_whenever/task/definition.rb

```diff
@@ -13,7 +14,12 @@ def name
       end
 
       def arn
-        definition&.task_definition_arn
+        arn = definition&.task_definition_arn
+        if family_with_revision?
+          arn
+        else
+          remove_revision(arn)
```

Comment:
By removing the revision here, the task definition will always use the latest version, not the latest version at the time it was updated. This means that, unlike before, the latest task definition is referenced without the user having to explicitly update it.

It's worth considering if this change is acceptable to current users, but probably no problem. I believe most users update their scheduled tasks when they update their task definitions.